components/spi/vault/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: spi-vault
+namePrefix: spi-
```

Comment:
be careful, this will add prefix to your resourcesSchema so then https://github.com/redhat-appstudio/service-provider-integration-operator/blob/main/config/kcp/apiexport_spi.yaml#L13 will not be working